src/interfaces/IBaseCurve.sol

```diff
@@ -82,6 +82,6 @@ interface IBaseCurve {
 
     /// @notice Get the current price of a share
     /// @param totalShares Total quantity of shares already awarded by the curve
-    /// @return sharePrice The current price of a share
```

Comment:
3.6 - currentSharePrice and currentSharePriceCurve Return Price With Orders of Magnitude Inflated by generalConfig.decimalPrecision

We updated comments and code to always produce share price reflected by a 1e18 scale